BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+
+**Matthias** - :  And yeah, so I pointed to him last time a few things to do regarding the proposal of just the editorial stuff, putting a number, putting the readme in the correct folder. He did all of that and he already marked it as active as I suggested, just because as I said, it's an informational CIP explaining the terminology picked by the core team. It's from the core team itself and it's also recording or explaining things from the past six eras or so. So yeah, there is no point going to things like propose, and then active, and then whatever, unless people agree with that. Okay. 
+
+**Sebastien** -, and your opinion maybe, if you are here.
+
+**Sebastien** -:  I joined probably through the conversation, so I'm missing the beginning.
+
+**Matthias** - :  Yeah. Okay, sorry. We are talking about CIP 59, the terminologies surrounding co-features, which was proposed by Jared some time ago to clarify a bit of the terminology around the co-features. And I was saying, this one could probably just go as active right away because it's just explaining what's in there.
+
+**Sebastien** -:  Yeah, I think so. I think that makes sense. I read through it and everything, and it made sense to me. So I don't mind both merging and marking it is actually the right away.
+
+**Matthias** - :  We have a question. Oh sorry, that's some other stuff. So yeah okay, let's do that. I think we have reviewed it. I would just add my review now, that's all the... Yeah, All change has been made. Happy to approve. Happy to merge. Okay. Next one, on chain token metadata standard which has been reviewed for a while. We had a few remark questions. A lot of people were debating stuff in particular. The security of that, I think Michael Peyton Jones is usually against anything that is put on chain and it is based on minting transactions because indeed it has security issues or implications.
+
+**Matthias** - :  If we start heavily relying on people providing the metadata being the same as people minting the tokens, which has been generally the case in Cardano, but it's not a given. So I think he has a fair point. But yeah, the proposal has been eight all ever since, I think. It hasn't really moved, we are waiting for the author to come back to it. But yeah, I'm not sure if there is much to say on this one. Yeah. No remark, no question. I guess we can just leave it there.
```

Comment:
```suggestion
**Matthias** - :  If we start heavily relying on people providing the metadata being the same as people minting the tokens, which has been generally the case in Cardano, but it's not a given. So I think he has a fair point. But yeah, the proposal has been stalled ever since, I think. It hasn't really moved, we are waiting for the author to come back to it. But yeah, I'm not sure if there is much to say on this one. Yeah. No remark, no question. I guess we can just leave it there.
```